apps/examples/reboot_reason_test/reboot_reason_main.c

```diff
@@ -141,6 +142,14 @@ static void memory_alloc_fail_test(void)
 }
 #endif
 
+static void prefetch_abort_test(void)
+{
+	/* The address allocated from heap can't be code. This will lead prefetch abort. */
+	int *ptr = (int *)malloc(1024);
```

Comment:
To use malloc, please remove the conditional at line 59 for stdlib.h.

Reply:
Modified. Thanks.